client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -554,6 +559,51 @@ function canItemBeDeleted( item: ResponseCartProduct ): boolean {
 	return ! itemTypesThatCannotBeDeleted.includes( item.product_slug );
 }
 
+function JetpackSearchMeta( { product }: { product: ResponseCartProduct } ): JSX.Element {
+	return (
+		<>
+			<ProductTier product={ product } />
+			<RenewalFrequency product={ product } />
+		</>
+	);
+}
+
+function ProductTier( { product }: { product: ResponseCartProduct } ): JSX.Element | null {
+	const translate = useTranslate();
+	const priceTierList = useSelector( ( state ) =>
+		getProductPriceTierList( state, product.product_slug )
+	);
+
+	if ( isJetpackSearch( product ) && product.current_quantity ) {
+		const tierMaximum = getPriceTierForUnits( priceTierList, product.current_quantity )
+			?.maximum_units;
+		if ( tierMaximum ) {
+			return (
+				<LineItemMeta>
+					{ translate( 'Up to %(tierMaximum)s records', { args: { tierMaximum } } ) }
```

Comment:
This won't display anything for the "more than 1,000,000 records" tier?  You could add something to handle that if you want, or not worry about it much because the number of sites that run into that is presumably small.

Reply:
Oh, good point.